src/lib/index.tsx

```diff
@@ -68,6 +69,7 @@ const CopyMailTo = ({
   containerStyles?: React.CSSProperties;
   tooltipStyles?: React.CSSProperties;
   anchorStyles?: React.CSSProperties;
+  tooltipPosition?: string;
```

Comment:
Instead of defining this property as a string use [TypeScript String Literals](https://www.typescriptlang.org/docs/handbook/literal-types.html#string-literal-types) to define the types of value this property can have